src/app/modules/calendar/calendar.module.ts

```diff
@@ -0,0 +1,30 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { CalendarComponent } from './calendar.component';
+import { BrowserModule } from '@angular/platform-browser';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import {MatDatepickerModule} from '@angular/material/datepicker';
```

Comment:
i'm being really picky & ocd here, but can you add spaces inside of the curly braces here and on line 9?